driver-core/src/main/com/mongodb/internal/connection/DefaultConnectionPool.java

```diff
@@ -560,7 +560,6 @@ public void onResult(final ResponseBuffers result, final Throwable t) {
 
         @Override
         public ConnectionDescription getDescription() {
-            isTrue("open", !isClosed.get());
```

Comment:
This causes lots of errors that leak to the logs, even if they are just side effects and have no practical implication:  Search for state should be: open in any evergreen log - even if the build was green:
eg: https://evergreen.mongodb.com/task_log_raw/mongo_java_driver_tests_jdk_secure__version~latest_os~linux_topology~standalone_auth~auth_ssl~ssl_jdk~jdk11_test_dac03bc270f733fef527ef12292f1bd3b05b8b86_20_03_11_21_48_49/0?type=T&text=true

Reply:
Ack